src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs

```diff
@@ -7,7 +7,7 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.Runtime;
 
-namespace Microsoft.ML.Transforms
+namespace Microsoft.ML.Transforms.Onnx
```

Comment:
all other extensions seem to be in the Microsoft.ML namespace. Let's leave this and the Resnet's there too.

Reply:
Signing off, but please move the extensions.

Reply:
thanks, done.